### ISSUE TITLE
[en] salutation comma

### ISSUE THREAD

**Issue by tiff** [languagetool-org/languagetool#1702]
When starting an email with:

* Hi Christopher
* Hello Christopher
* ...

We should suggest adding a comma afterwards.

Same with ending an email:

* Kind regards
* Regards
* ...

**Comment by tiff** [comment]
@danielnaber do we have something like a "line-start" and "line-end" token?

**Comment by danielnaber** [comment]
@tiff no, we don't have such a concept, but maybe the new line is interpreted as another paragraph?

**Comment by tiff** [comment]
@danielnaber no it isn't 

test text:

```
Hello Daniel
How are you?

Kind regards
Christopher
```

**Comment by danielnaber** [comment]
Can't we have a rule that matches "Hello <name>" when not followed by a comma or anything? Issue would be that user might type "Hello ..." and then stop -> we'd already show an error I guess.

**Comment by tiff** [comment]
This rule would trigger for every chat message: "Hi Daniel"
